### DISCUSSION TITLE
信息卡片中的角色卡片可以稍微宽一丢丢

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#729] [General]
![image](https://user-images.githubusercontent.com/48074861/158983108-b732530b-294e-46d6-819d-7c39c77a26c9.jpeg)
如图，温迪的终末换行了，稍微宽一点就不会逼死强迫症了。

> 另，入坑一周年纪念

**Comment by Xm798** [comment]
@Mark9804

**Comment by mark9804** [comment]
待会我把炼字给删了就行了

另外你的心没有，应该是没这字体
<img width="263" alt="image" src="https://user-images.githubusercontent.com/9006264/158987340-8e0aa757-49bf-4bcb-893d-14e0d8dea9e4.png">

**Reply by Xm798** [reply]
这是个什么字体

**Reply by mark9804** [reply]
呃，理论上是会掉落到你系统当中的等宽字体上的

https://github.com/Arondight/Adachi-BOT/blob/6764d6e5700969c50e5497a67d673184b2577733/src/views/component/card-8/card-8.css#L30-L32

可能你系统也没有等宽字体？那我待会改成 sans-serif 好了，这个不可能没有

**Reply by mark9804** [reply]
好了

**Reply by Xm798** [reply]
好耶
